tools/rosetta/lib/server/server.go

```diff
@@ -45,7 +45,7 @@ type Server struct {
 
 func (h Server) Start() error {
 	h.logger.Info(fmt.Sprintf("Rosetta server listening on add %s", h.addr))
-	return http.ListenAndServe(h.addr, h.h) //nolint:gosec
+	return http.ListenAndServe(h.addr, h.h) //nolint:gosec // we don't care about the security of the server, users should proxy
```

Comment:
```suggestion
	return http.ListenAndServe(h.addr, h.h) //nolint:gosec // users are recommended to operate a proxy in front of this server 
```

lets put a better comment here